### DISCUSSION TITLE
ManageMeals is now self-hostable

### DISCUSSION THREAD

**Discussion by hilmarp** [managemeals/manage-meals-api#3] [Announcements]
See the [wiki](https://github.com/managemeals/manage-meals-api/wiki/Self%E2%80%90hosting).

**Comment by thebrokennomad** [comment]
Register/Login doesn't work after running the docker installation.

$ docker compose up -d
[+] Running 57/57
 ✔ manage-meals-web Pulled                                                                                                                           178.6s 
 ✔ mongo-express Pulled                                                                                                                               65.1s 
 ✔ mongo Pulled                                                                                                                                      182.2s 
 ✔ recipe-scraper Pulled                                                                                                                              70.0s 
 ✔ manage-meals-api Pulled                                                                                                                           132.1s 
 ✔ typesense Pulled                                                                                                                                  197.8s 
 ✔ valkey Pulled                                                                                                                                      65.2s 
 ✔ search-sync Pulled                                                                                                                                 54.9s 

[+] Running 11/11
 ✔ Network managemeals_default               Created                                                                                                   0.1s 
 ✔ Volume managemeals_mongo-data             Created                                                                                                   0.0s 
 ✔ Volume managemeals_typesense-data         Created                                                                                                   0.0s 
 ✔ Container managemeals-valkey-1            Started                                                                                                   2.2s 
 ✔ Container managemeals-recipe-scraper-1    Started                                                                                                   2.1s 
 ✔ Container managemeals-mongo-1             Started                                                                                                   1.9s 
 ✔ Container managemeals-typesense-1         Started                                                                                                   2.3s 
 ✔ Container managemeals-mongo-express-1     Started                                                                                                   2.1s 
 ✔ Container managemeals-search-sync-1       Started                                                                                                   2.3s 
 ✔ Container managemeals-manage-meals-api-1  Started                                                                                                   2.7s 
 ✔ Container managemeals-manage-meals-web-1  Started                                                                                                   3.5s 

$ docker ps
CONTAINER ID   IMAGE                                                 COMMAND                  CREATED          STATUS                            PORTS                                         NAMES
ad75cd2079d8   ghcr.io/managemeals/manage-meals-web:latest           "docker-entrypoint.s…"   20 minutes ago   Up 20 minutes                     0.0.0.0:3001->3000/tcp, [::]:3001->3000/tcp   managemeals-manage-meals-web-1
dbeff682995b   ghcr.io/managemeals/manage-meals-api:latest           "docker-entrypoint.s…"   20 minutes ago   Up 12 seconds                     0.0.0.0:3000->3000/tcp, [::]:3000->3000/tcp   managemeals-manage-meals-api-1
97ab59118572   mongo-express                                         "/sbin/tini -- /dock…"   20 minutes ago   Up 17 seconds                     0.0.0.0:8081->8081/tcp, [::]:8081->8081/tcp   managemeals-mongo-express-1
b22bf5f412dc   ghcr.io/managemeals/manage-meals-search-sync:latest   "docker-entrypoint.s…"   20 minutes ago   Up 20 seconds                                                                   managemeals-search-sync-1
4e29d3756772   mongo:7                                               "docker-entrypoint.s…"   20 minutes ago   Restarting (132) 22 seconds ago                                                 managemeals-mongo-1
f00d3b83993e   ghcr.io/managemeals/manage-meals-scraper:latest       "gunicorn -w 1 -b 0.…"   20 minutes ago   Up 20 minutes                     0.0.0.0:8000->8000/tcp, [::]:8000->8000/tcp   managemeals-recipe-scraper-1
9c845794c6f9   typesense/typesense:28.0                              "/opt/typesense-serv…"   20 minutes ago   Up 20 minutes                     0.0.0.0:8008->8108/tcp, [::]:8008->8108/tcp   managemeals-typesense-1
f1286fd0c583   valkey/valkey:8                                       "docker-entrypoint.s…"   20 minutes ago   Up 20 minutes                     0.0.0.0:6379->6379/tcp, [::]:6379->6379/tcp   managemeals-valkey-1
